### PR TITLE
Adding collectd package installer script

### PR DESCRIPTION
**Pull request by sschemel**

### CODE REVIEW

File: collectd-package-install.sh [hunk 1]
```diff
@@ -0,0 +1,312 @@
+#! /bin/bash
+#exec 3>&1 1>> sfx_install_collectd.log 2>&1
```
Comment (tedoc2000) [inline collectd-package-install.sh]: Don't check in commented out code

File: collectd-package-install.sh [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+centos_7_rpm="SignalFx-RPMs-centos-7-release-1.0-0.noarch.rpm"
+centos_6_rpm="SignalFx-RPMs-centos-6-release-1.0-0.noarch.rpm"
+centos_5_rpm="SignalFx-RPMs-centos-5-release-1.0-0.noarch.rpm"
+aws_linux_2014_09_rpm="SignalFx-RPMs-AWS_EC2_Linux_2014_09-release-1.0-0.noarch.rpm"
+aws_linux_2015_03_rpm="SignalFx-RPMs-AWS_EC2_Linux_2015_03-release-1.0-0.noarch.rpm"
+
+#determine hostOS
+hostOS=$(cat /etc/*-release | grep PRETTY_NAME | grep -o '".*"' | sed 's/"//g' | sed -e 's/([^()]*)//g' | sed -e 's/[[:space:]]*$//') #for newer versions of linux
+hostOS_2=$(cat /etc/redhat-release | head -c 16) #older versions of RPM based linux that don't have version in PRETTY_NAME format
+#hostOS_3=$(cat /etc/*-release | grep DISTRIB_DESCRIPTION | grep -o '".*"' | sed 's/"//g' | sed -e 's/([^()]*)//g' | sed -e 's/[[:space:]]*$//' | head -c 12)
```
Comment (tedoc2000) [inline collectd-package-install.sh]: Same here.

File: collectd-package-install.sh [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+			needed_rpm_name=$aws_linux_2015_03_rpm
+		;;
+		*)
+		;;
+	esac
+}
+
+get_os_input() 
+{
+	#echo $selection #check for currently value of selection
```
Comment (tedoc2000) [inline collectd-package-install.sh]: Fix this so it is only a comment

**Comment by tedoc2000** [comment]
Can you switch to using printf

File: collectd-package-install.sh [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+		curl -sSL https://dl.signalfx.com/collectd-simple | sudo bash -s -- -t $api_token
+	fi
+
+}
+#aggregatedhost_collectd() #url to assume hostname. Asks for username:password
+
+install_success()
+{
+	echo "
+Install is now compelete and you can view your metrics at app.signalfx.com.
```
Comment (udaysagar2177) [inline collectd-package-install.sh]: complete - typo

**Comment by mpetazzoni** [comment]
@StratenFX I'd like us to be very careful about code quality and documentation in any code that we release to the public. The quality of these little scripts for example has a big impact on the perception of the quality of engineering at SignalFx -- especially since they are one of the first contact our users/customers will have with software we provide.

File: collectd-package-install.sh [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+
+#rpm file variables
+centos_7_rpm="SignalFx-RPMs-centos-7-release-1.0-0.noarch.rpm"
+centos_6_rpm="SignalFx-RPMs-centos-6-release-1.0-0.noarch.rpm"
+centos_5_rpm="SignalFx-RPMs-centos-5-release-1.0-0.noarch.rpm"
+aws_linux_2014_09_rpm="SignalFx-RPMs-AWS_EC2_Linux_2014_09-release-1.0-0.noarch.rpm"
+aws_linux_2015_03_rpm="SignalFx-RPMs-AWS_EC2_Linux_2015_03-release-1.0-0.noarch.rpm"
+
+#download location variables
+centos_7="https://dl.signalfx.com/rpms/SignalFx-rpms/release/$centos_5_rpm"
```
Comment (tedoc2000) [inline collectd-package-install.sh]: can you put ${} around variable names. It just makes them super clear.
e.g: 
centos_7="https://dl.signalfx.com/rpms/SignalFx-rpms/release/${centos_5_rpm}"

**Comment by sschemel** [comment]
Got the 'ok' from Tedo

**Pull request merged**